src/org/omegat/filters4/xml/openxml/MsOfficeFileFilter.java

```diff
@@ -239,7 +239,7 @@ protected java.util.Comparator<ZipEntry> getEntryComparator() {
 
     @Override
     public Instance[] getDefaultInstances() {
-        return new Instance[] { new Instance("*.doc?"), new Instance("*.dotx"), new Instance("*.xls?"),
+        return new Instance[] { new Instance("*.doc?"), new Instance("*.dotx"), //new Instance("*.xls?"),
```

Comment:
It  is commented out because of a known bug, I think it is better to add a reference of the bug ticket.
```suggestion
        // disable a link for MS Excel files: https://sourceforge.net/p/omegat/bugs/1260
        return new Instance[] { new Instance("*.doc?"), new Instance("*.dotx"), //new Instance("*.xls?"),
```

Reply:
There was a reference to the ticket in the commit message, but ok, no problem for me